tools/harness-thci/OpenThread.py

```diff
@@ -1847,7 +1846,7 @@ def getGUA(self, filterByPrefix=None):
                     if fullIp.startswith(filterByPrefix):
                         return fullIp
                 print 'no global address matched'
-                return None
+                return str(globalAddrs[0])
```

Comment:
we may better to check `globalAddrs` is not empty after ` globalAddrs = self.getGlobal()` for robust?

Reply:
may enhance in separate PR